packages/api-sdk/src/medical/client/metriport.ts

```diff
@@ -713,19 +713,20 @@ export class MetriportMedicalApi {
    * Verifies the signature of a webhook request.
    * Refer to Metriport's documentation for more details: https://docs.metriport.com/medical-api/more-info/webhooks.
    *
-   * @param wh_key - your webhook key.
-   * @param req.body - the body of the webhook request.
+   * @param key - your webhook key.
+   * @param body - the body of the webhook request, as string.
    * @param signature - the signature obtained from the webhook request header.
    *
    * @returns True if the signature is verified, false otherwise.
    */
-  static verifyWebhookSignature(wh_key: string, reqBody: string, signature: string): boolean {
-    const signatureAsString = String(signature);
-    const receivedHash = crypto
-      .createHmac("sha256", wh_key)
-      .update(JSON.stringify(reqBody))
-      .digest("hex");
-    return receivedHash === signatureAsString;
+  static verifyWebhookSignature(key: string, body: string, signature: string): boolean {
+    const normalizedBody = typeof body === "string" ? body : JSON.stringify(body);
```

Comment:
so what the comment on the security notice meant here, was that the act of calling JSON.stringify has a normalizing effect which is bad for security. 
If payload: { A: "C"} and {A: "C"    } are different but when you call stringify they become the same thing, then that means there are multiple different payloads that can result in the same HMAC. So The guy was saying that we should be calling stringify and should just be calling verify on the raw body. 

This is fine we just need to verify that the webhook validation still works in that case.